rusk-abi/src/host.rs

```diff
@@ -7,54 +7,48 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use canonical::{ByteSource, Canon, InvalidEncoding, Store};
+use canonical::{Canon, CanonError, Source};
 use dusk_abi::{HostModule, Query, ReturnValue};
 use dusk_bls12_381::BlsScalar;
 use dusk_bytes::DeserializableSlice;
 use dusk_pki::PublicKey;
 use dusk_plonk::circuit::{self, VerifierData};
 use dusk_plonk::prelude::*;
-use schnorr::Signature;
+use dusk_schnorr::Signature;
 
 use crate::{PublicInput, RuskModule};
 
-impl<S> RuskModule<S>
-where
-    S: Store,
-{
-    pub fn new(store: S, pp: &'static PublicParameters) -> Self {
-        RuskModule { store, pp }
+impl RuskModule {
+    pub fn new(pp: &'static PublicParameters) -> Self {
+        RuskModule { pp }
```

Comment:
Not for this release. But we could make this much easier to load by just storing `&'static OpeningKey` which is the part that is indeed neded from the `PublicParameters` in order to verify a proof. 

Maybe filing an issue is worth it.

Reply:
Filed https://github.com/dusk-network/rusk/issues/289